lib/xenops_server.ml

```diff
@@ -949,12 +959,13 @@ let rec atomics_of_operation = function
     ; List.map (fun vif -> VIF_set_active (vif.Vif.id, true)) vifs
     ; List.map (fun vif -> VIF_plug vif.Vif.id) vifs
     ; List.map (fun vgpu -> VGPU_set_active (vgpu.Vgpu.id, true)) vgpus
+    ; List.map (fun pci -> PCI_plug(pci.Pci.id, false)) pcis_sriov
```

Comment:
We also have network SRIOV devices, should this be called something different? `pcis_sriov_nvidia`?